apps/studio/data/database-triggers/database-trigger-delete-mutation.ts

```diff
@@ -19,20 +20,16 @@ export async function deleteDatabaseTrigger({
   projectRef,
   connectionString,
 }: DatabaseTriggerDeleteVariables) {
-  let headers = new Headers()
-  if (connectionString) headers.set('x-connection-encrypted', connectionString)
+  const { sql } = pgMeta.triggers.remove({ id })
```

Comment:
Please update the underlying query to match pgmeta https://github.com/supabase/postgres-meta/blob/master/src/lib/PostgresMetaTriggers.ts#L234

Reply:
Updated in https://github.com/supabase/supabase/pull/35667/commits/37c844693eb3f81dd59a5c45da60b0d4b92b8ff8